project-base/storefront/utils/productLists/useProductList.ts

```diff
@@ -30,7 +31,7 @@ export const useProductList = (
     const [, TypeRemoveProductFromListMutation] = useRemoveProductFromListMutation();
     const [, removeListMutation] = useRemoveProductListMutation();
 
-    const [{ data: productListData, fetching: isProductListFetching }] = useProductListQuery({
+    const [{ data: productListData, fetching }] = useProductListQuery({
```

Comment:
if you have to rename it, please rename it to `isProductListQueryFetching`, or something like that. We have renamed this just recently to more descriptive names and would definitely like to keep it.

Reply:
Sure